src/screens/ImportMethod/ImportMethod.js

```diff
@@ -0,0 +1,87 @@
+/**
+ * Copyright 2017–2018, LaborX PTY
+ * Licensed under the AGPL Version 3 license.
+ */
+
+import React, { PureComponent } from 'react'
+import {
+  Image,
+  Text,
+  TouchableOpacity,
+  View,
+} from 'react-native'
+import textConstants from '../../locales/en'
+import PropTypes from 'prop-types'
```

Comment:
Code style:
All imports with absolute paths must be placed above all other imports with relative paths ('../')